<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_window</class>
 <widget class="QMainWindow" name="Main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SES_plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <widget class="QLineEdit" name="lineEdit_dir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>144</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_scan_status">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>29</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Idle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unzip">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>39</y>
      <width>62</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>UnZip</string>
    </property>
   </widget>
   <widget class="QLabel" name="fixed_label_energy_width">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_file">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>38</y>
      <width>144</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fixed_label_energy_center">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="icon">
    <iconset>
     <normaloff>../icons/arrow-circle-225.png</normaloff>../icons/arrow-circle-225.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
